docs/run-operator-locally.md

```diff
@@ -4,26 +4,43 @@ This document contains a quickstart guide to build and running (+debugging) the
   - [contributing.md](contributing.md)
   - [build_operator_locally.md](build_operator_locally.md)
-  - if above has been configured there should be either:
+  - If above has been configured there should be either:
     - `$HOME/.kube/config`
     - `KUBECONFIG` environment variable pointing at a file
     - **Note**: either of these are necessary to be able to run the operator locally
-- have a folder `.community-operator-dev`
-- this guides uses intellij as an example. Running any other way should be very similar.
+- Have a folder `.community-operator-dev`
 ## Goals
-- runs the operator locally as a binary (optionally in debug mode) in with telepresence
-- runs e2e tests locally 
-  - **Note:** 
-    - if you plan to run the e2e tests, there are sub-steps that will install the following helm-chart: [operator.yaml](helm-charts/charts/community-operator/templates/operator.yaml)
-    - by default, the template chart contains and the operator with `1` replica. This will bite with our local running operator. With this in mind the solution is to set the number to `0`
-
-## Steps for Intellij
-- run the make target creating env file `make generate-env-file` 
-  - which creates/updates an env file `local-test.env` in .community-operator-dev/
-- use said env file in intellij while running the main of the operator: `cmd/manager/main.go`
+- Run the operator locally as a binary (optionally in debug mode) in command line or in an IDE
+- Run e2e tests locally
+
+## Running The Operator locally - Intellij
+1. Run the make target creating env file `make generate-env-file`, which creates/updates an env file `local-test.env` in .community-operator-dev/
+2. Use local-test.env file in IntelliJ while running the main of the operator: `cmd/manager/main.go`
 ![img1](images/intellij-run-env.png)
 ![img2](images/intellij-run-env-2.png)
-- with the env setup, one can build and run/debug the operator locally and things should work OOTB if the prerequisites have been fulfilled.
+
+
+## Running The Operator locally - Shell
+1. Use the dedicated make target which exports the needed environment variables and builds & runs the operator binary
+
+```sh
+make run 
+```
+
+2.  For debugging one can use the following make target, which uses [dlv](https://github.com/go-delve/delve):
+
+```sh
+make debug
+```
+
+## Running e2e tests with the local operator
+- Our [e2e tests](../test/e2e), contains sub-steps that will install the following helm-chart: [operator.yaml](../helm-charts/charts/community-operator/templates/operator.yaml)
+- By default, the template chart sets the number of operator replicas to `1`. This will clash with our local running operator. With this in mind the solution is to set the replicas number to `0` temporarily.
+- Follow the guide on how to run `e2e` tests as described in our [contributing.md](contributing.md), for instance:
+
+```sh
+make e2e-telepresence test=<test-name>
```

Comment:
This will deploy a pod and this guide is about not deploying pods for the operator and testing. Let's add a point about running specific e2e test locally from cmdline and IntelliJ.

Reply:
I'm mistaken. It will run locally!